routes/contribute/pad/index.js

```diff
@@ -221,9 +221,9 @@ module.exports = (req, res) => {
 
 					if (id) data = await datastructures.legacy.publishablepad({ connection: t, data });
 
-					const excerpt = data.status > 2 ? { title: data.title, txt: parsers.getTxt(data)[0], img: { src: parsers.getImg(data)[0], width: 300, height: 200 }, p: true } : null
+					const excerpt = (data && data.status && data.status > 2) ? { title: data.title, txt: parsers.getTxt(data)[0], img: { src: parsers.getImg(data)[0], width: 300, height: 200 }, p: true } : null
```

Comment:
`data` here can be uninitialized in certain circumstances (e.g., a new pad) -- we can make excerpt null if data happens to not have any information